packages/tasit-contracts/src/scripts/populateDecentralandContracts.js

```diff
@@ -78,263 +213,184 @@ const createEstate = async (estateContract, landContract, estate, wallet) => {
+    if (!metadata) metadata = "";
+    return { x, y, metadata };
+  });
 
-  const allParcels = [
-    // Estate: all road adjacent parcels
-    { x: -30, y: -105, metadata: `` },
-    { x: -31, y: -105, metadata: `` },
-    { x: -29, y: -105, metadata: `` },
-    { x: -30, y: -104, metadata: `` },
-    { x: -29, y: -104, metadata: `` },
-
-    // Estate: Forest / university estate
-    { x: 3, y: 141, metadata: `` },
-    { x: 2, y: 141, metadata: `` },
-
-    // Estate: Down-Town Bridge
-    { x: -39, y: 30, metadata: `` },
-    { x: -39, y: 31, metadata: `` },
-
-    // Estate: Decentraland University Underpass
-    { x: -47, y: 124, metadata: `` },
-    { x: -48, y: 124, metadata: `` },
-
-    // Estate: Villa Beau Soleil H
-    { x: 39, y: -114, metadata: `Letter N` },
-    { x: 39, y: -113, metadata: `Letter "N"` },
-
-    // Unique parcels
-    { x: -20, y: 36, metadata: `Premium Downtown,road adjacent,central area.` },
-    { x: -61, y: 125, metadata: `Vegas/Univeristy` },
-    { x: 141, y: -122, metadata: `dePeets Place 6` },
-    { x: -150, y: 22, metadata: `Fashion District X` },
-    { x: -150, y: 23, metadata: `District X Fashion Sandwich` },
-    {
-      x: -7,
-      y: -110,
-      metadata: `On a junction right opposite two large estates :O)`,
-    },
-  ];
-
-  const allEstates = [
-    {
-      metadata: `all road adjacent parcels`,
-      parcels: allParcels.slice(0, 5),
-    },
-    {
-      metadata: `Forest / university estate`,
-      parcels: allParcels.slice(5, 7),
-    },
-    {
-      metadata: `Down-Town Bridge`,
-      parcels: allParcels.slice(7, 9),
-    },
-    {
-      metadata: `Decentraland University Underpass`,
-      parcels: allParcels.slice(9, 11),
-    },
-    {
-      metadata: `Estate: Villa Beau Soleil H`,
-      parcels: allParcels.slice(11, 13),
-    },
-  ];
+  return parcels;
+};
 
+(async () => {
   try {
-    console.log("Creating parcels...");
-    await createMultipleParcels(
-      landContract,
-      allParcels,
-      sellerAddress,
-      minterWallet
-    );
+    // Fund Gnosis Safe wallet with Mana tokens and ethers
+    await etherFaucet(provider, minterWallet, GNOSIS_SAFE_ADDRESS, TEN);
+    await erc20Faucet(manaContract, minterWallet, GNOSIS_SAFE_ADDRESS, BILLION);
 
-    const allParcelsIds = allParcels.map(async parcel => {
-      const { x, y } = parcel;
-      return await landContract.encodeTokenId(x, y);
-    });
+    const parcelsFromAPI = await getParcelsFromAPI();
+    const estatesFromAPI = await getEstatesFromAPI();
 
-    // Update parcels with metadata
-    console.log("Updating parcels with metadata...");
-    landContract.setWallet(sellerWallet);
-    for (let parcel of allParcels) {
-      const { x, y, metadata: parcelName } = parcel;
-      const updateAction = landContract.updateLandData(x, y, parcelName);
-      await updateAction.waitForNonceToUpdate();
-    }
+    const estatesParcels = extractParcelsFromEstates(estatesFromAPI);
```

Comment:
Why is there any need to extract these parcels instead of just dealing with the estates directly?

Reply:
Is it because sometimes individual parcels from within the estate are also for sale? I don't know if that's necessarily a problem

Reply:
A) The parcels creation function receives a parcels list and I'm extracting those to call that function once. On the other hand, I can call several times (one time for each estate).

B) I'm removing duplicated parcels from an estate that already was getting from parcels API. On the other hand, the parcel creation function is already ignoring failed creation.

Reply:
A) Ah yes, I forgot we would need to create the underlying parcels that belong to the estates in the test env. Makes a lot of sense now, though.

B) Ah ok, so it's just to avoid creating them twice. It's still nice from a code understandability perspective not to count on failed creation to keep them from being created. So I like the way you've done this.

C) We still list the parcels and estates for sale separately if a parcel is for sale that's also in an estate that's for sale, right? This was the topic I was originally thinking about when creating this thread.

This all makes sense to me now, and I think we can resolve it.

Reply:
A and B) :+1: 
C) I'm almost sure that if an estate is for sale, none of its parcels could have a sell order too.

Ok, resolving this.

Reply:
C) I think part of the reason for fingerprinting estates is because that is possible. But maybe it's disallowed on the front end.